lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -583,6 +547,13 @@ bool TypeLoweringVisitor::lowerProducer(
   if (!peelType(srcType, fieldTypes, aggregatePreservationMode))
     return false;
 
+  // If an aggregate value has a symbol, bail out.
+  if (op->hasAttr(cache.innerSymAttr)) {
+    op->emitError() << "LowerTypes cannot lower symbols on aggregates";
```

Comment:
Nit: this error message kind of indicates that `LowerTypes` doesn't support lowering symbols when we want to indicate that it is (currently) illegal to pass IR that has symbols into `LowerTypes`. Maybe: `"has a symbol, but no symbols may exist on aggregates passed through LowerTypes"`.